### ISSUE TITLE
[Service Integration]: OriginTrail Node Integration

### ISSUE THREAD

**Issue by daGscheid** [stereum-dev/ethereum-node#1424]
### Github Link for Service

https://github.com/OriginTrail

### Tell us something about the service!

Requested by our dear user CosmiCloud, we should think about branching out into the Web3 Asset Space with OriginTrail - 

https://origintrail.io/technology/run-a-node

OriginTrail Decentralized Knowledge Graph (DKG) presents a global, open data structure comprised of interlinked knowledge assets. It is hosted on an open, permissionless OriginTrail Decentralized Network (ODN) which combines blockchain with knowledge graph technology. 

A knowledge asset is an ownable container for your valuable information that can describe any physical thing, documents, abstract concepts, numbers and strings. It lives in a decentralized knowledge graph which makes it easily discoverable, while your information integrity and knowledge asset ownership are safeguarded by the underlying blockchain.
More precisely, a Knowledge Asset is a web resource identified by Uniform Asset Locators (or UAL, which is an extension of the traditional URL), consisting of
the asset graph, which contains the knowledge asset data, represented in RDF , stored on the DKG (and not on the blockchain)
Immutability proofs & ownership record: consisting of asset graph state proofs and ownership record, represented using a non-fungible token stored on a blockchain.

Asset graphs are formed of assertions, which represent asset content states. An assertion is represented using the n-quads serialization (stored on the DKG) and a cryptographic fingerprint used for assertion verification (n-quads graph Merkle root, stored immutably on the blockchain). Each assertion can be independently verified for integrity, by recomputing the cryptographic fingerprint by the verifier and comparing if the computed result matches with the corresponding blockchain fingerprint record.
Knowledge Assets can contain both public and private data - the public assertion data being replicated on the OriginTrail Decentralized Network and publicly available, and private assertion data contained within the private domain of the asset owner (e.g. OriginTrail Node run by the asset owner, such as a person or company).
In summary, a knowledge asset is a combination of an NFT record and a semantic record. Using the dkg.js SDK you will be able to perform CRUT (create, read, update, transfer) operations with Knowledge assets, which will be explained in further detail below.

### Documentation Link for Service

https://docs.origintrail.io/decentralized-knowledge-graph-layer-2/introduction-to-dkg-v6-start-here

### Acknowledgment

- [X] I looked through the existing requests of this project
